src/components/ui/toolbar/index.js

```diff
@@ -130,6 +130,20 @@ const toolbar = (services) => {
             alert(localeService.t('nodocselected'));
         }
     };
+    const _triggerModal_2 = (event, actionFn) => {
```

Comment:
duplicated code, better to 
- add an argument "alertIfNoDocSelected" to _triggerModal() with default value true
or better
- add an argument "options" (object to encapsulate extensible options)
then
- fix every call (or code default options / no bc break)